.github/labeler.yml

```
@@ -33,3 +33,94 @@ config-change:
 template:
   - changed-files:
       - any-glob-to-any-file: src/main/resources/templates/**/*
+
+# module labels
+assessment:
```

```suggestion
"module:assessment":
```
Would it make sense to group the labels with a common prefix like the current `Component:` labels?

I like that the current Component labels are easily discoverable since they all use the same prefix and colour.

---

1. I think this is a very subjective topic. I personally dislike the `component` prefix, because it makes it unnecessarily complicated to quick search for them.

2. This is future work. We're currently in a discussion if/when the client will follow the same modules. Additionally, `component` is only used in issues. A section that should also be accessible for users that are not necessarily participating in the development of Artemis, while the new modules aim to improve the workflow for our developers and should be of no concern for most users (except administrators).